.github/workflows/project-codeflare-release.yml

```diff
@@ -41,12 +41,18 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Check if MCAD release does exist
+      run: |
+        status_code=$(curl -s -o /dev/null -w "%{http_code}" https://github.com/project-codeflare/multi-cluster-app-dispatcher/releases/tag/${{ github.event.inputs.mcad-version }})
```

Comment:
Have you considered using GH CLI?
https://github.com/project-codeflare/codeflare-operator/blob/main/.github/workflows/tag-and-build.yml#L65-L75

It is longer but may be more descriptive.
Also it would be good to notify users that the release already exists.